src/App.tsx

```diff
@@ -15,36 +15,85 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+export enum SortKeys {
+  ALPHABETICAL = 'Alphabetical',
+  LENGTH = 'Length',
+}
+
+export const App: FC = () => {
+  const [sort, setSort] = useState<string>('');
```

Comment:
```suggestion
  const [sort, setSort] = useState<SortType>(SortType.Default);
```
It's not a string, use corresponding type